connectors/aws-s3-sink/v3.0.2.1/aws-s3-sink.md

```diff
@@ -36,39 +36,62 @@ The prerequisites for connecting an AWS S3 sink connector to external systems in
 1. Create S3 buckets in AWS.
 2. Create the [AWS User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) and create `AccessKey`(Please record `AccessKey` and `SecretAccessKey`).
 3. Assign permissions to AWS User, and ensure they have the following permissions to the AWS S3. 
-```text
-- s3:AbortMultipartUpload
-- s3:CreateBucket
-- s3:PutObject
+```json
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Sid": "VisualEditor0",
+			"Effect": "Allow",
+			"Action": [
+				"s3:PutObject",
+				"s3:AbortMultipartUpload"
+			],
+			"Resource": "{Your bucket arn}/*"
+		}
+	]
+}
 ```
 
 ### 1. Create a connector
 
-Depending on the environment, there are several ways to create an AWS S3 sink connector:
-
-- [Create Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
-- [Create Connector with Function worker](https://pulsar.apache.org/docs/3.0.x/io-quickstart/).
-  Using this way requires you to download a **JAR** package to create a built-in or non-built-in connector. You can download the version you need from [here](https://github.com/streamnative/pulsar-io-cloud-storage/releases).
-- [Create Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
-  Using this way requires you to set the docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-cloud-storage/tags)
-
-No matter how you create an AWS S3 sink connector, the minimum configuration contains the following parameters.
-
-```yaml
-   configs:
-     accessKeyId: "{{Your access access key}}"
-     secretAccessKey: "{{Your secret access key}}"
-     provider: "s3v2"
-     bucket: "{{Your bucket name}}"
-     region: "{{Your AWS S3 region}}"
-     formatType: "json"  
-     partitionerType: "PARTITION" 
+The following command shows how to use [pulsarctl](https://github.com/streamnative/pulsarctl) to create a `builtin` connector. If you want to create a `non-builtin` connector,
+you need to replace `--sink-type cloud-storage-s3` with `--archive /path/to/pulsar-io-cloud-storage.nar`. You can find the button to download the `nar` package at the beginning of the document.
+
+{% callout title="For StreamNative Cloud User" type="note" %}
+If you are a StreamNative Cloud user, you need [set up your environment](https://docs.streamnative.io/docs/connector-setup) first.
+{% /callout %}
+
+```bash
+pulsarctl sinks create \
+  --sink-type cloud-storage-s3 \
+  --name aws-s3-sink \
+  --tenant public \
+  --namespace default \
+  --inputs "Your topic name" \
+  --parallelism 1 \
+  --sink-config \
+  '{
+    "accessKeyId": "Your access access key", 
+    "secretAccessKey": "Your secret access key",
```

Comment:
Need to fix this typo: https://github.com/streamnative/pulsar-io-aws-eventbridge/pull/178